modules/osd-create-cluster-ccs.adoc

```diff
@@ -0,0 +1,221 @@
+====
+.. Optional: You can select *Bypass AWS Service Control Policy (SCP) checks* to disable the SCP checks.
++
+[NOTE]
+====
+Some AWS SCPs will cause the installation to fail, even if you have the required permissions. Disabling the SCP checks allows an installation to proceed. The SCP will still be enforced even if the checks are bypassed.
+====
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+. Provide your GCP service account private key in JSON format. You can either click *Browse* to locate and attach a JSON file or add the details in the *Service account JSON* field.
+endif::osd-on-gcp[]
+
+. *Validate* your cloud provider account and then click *Next*.
+
+. On the *Cluster details* page, provide a name for your cluster and specify the cluster details:
+.. Add a *Cluster name*.
+.. Select a cluster version from the *Version* drop-down menu.
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. The option is enabled by default.
+.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption. With this option, the etcd key values are encrypted, but not the keys. The option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
++
+[NOTE]
+====
+By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
+====
+.. Optional: Select *Encrypt persistent volumes with customer keys* if you want to provide your own 
+ifdef::osd-on-aws[]
+AWS Key Management Service (KMS) key Amanzon Resource Name (ARN). 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+encryption keys through the Google Cloud Key Management Service. 
+endif::osd-on-gcp[]
+The key is used for encryption of persistent volumes in your cluster.
+.. Click *Next*.
+
+. On the *Default machine pool* page, select a *Compute node instance type* and a *Compute node count*. The number and types of nodes that are available depend on your {product-title} subscription. If you are using multiple availability zones, the compute node count is per zone.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type. The number and types of nodes available to you depend on your {product-title} subscription.
+====
+
+. Optional: Expand *Edit node labels* to add labels to your nodes. Click *Add label* to add more node labels.
```

Comment:
Lack of "Click Next button" step to go to next page

Reply:
Thank you for highlighting this. Resolved.